frontend/src/app/main/ui/workspace/sidebar/options/rows/color_row.scss

```diff
@@ -23,157 +23,161 @@
   gap: $s-2;
   border-radius: $s-8;
   background-color: var(--input-details-color);
-  height: $s-32;
-  width: 100%;
+  block-size: $s-32;
+  inline-size: 100%;
   flex-grow: 1;
 
-  .color-name-wrapper {
-    @extend .input-element;
-    flex-grow: 1;
-    width: 100%;
-    border-radius: $br-8 0 0 $br-8;
-    padding: 0;
-    margin-right: 0;
-    gap: $s-4;
-    input {
-      padding: 0;
-    }
-    .color-bullet-wrapper {
-      height: $s-28;
-      padding: 0 $s-2 0 $s-8;
-      border-radius: $br-8 0 0 $br-8;
+  &:hover {
+    .detach-btn,
+    .select-btn {
       background-color: transparent;
-      &:hover {
-        background-color: transparent;
+      svg {
+        stroke: var(--input-foreground-color-active);
```

Comment:
I know we have this kind of code all over the place, but I try to avoid using raw `svg` selectors with the aid of both CSS custom properties + adding a class to the SVG itself.

If you are curious you might check out the `pin-button` component ([cljs](https://github.com/penpot/penpot/blob/b426db133ddee98b7c9977247f92c59eca5085c0/frontend/src/app/main/ui/dashboard/pin_button.cljs#L18-L19) and [scss](https://github.com/penpot/penpot/blob/b426db133ddee98b7c9977247f92c59eca5085c0/frontend/src/app/main/ui/dashboard/pin_button.scss#L24-L27))

Reply:
Fixed. This SVG already has a class. I've updated the selector.